newIDE/app/src/MainFrame/index.js

```diff
@@ -207,6 +208,7 @@ export type State = {|
   editorTabs: EditorTabsState,
   snackMessage: string,
```

Comment:
You could try a `key` on the Snackbar to see if this component can animate when the message changes.

Reply:
I went for a different strategy, using a different method which closes the snackbar and reopens it after a short delay.
It looks pretty nice (see video)